nova/nova/internal_views/alphavantage.py

```diff
@@ -36,18 +34,19 @@ def do_request_daily(tr_eq):
         to_param = '&market='
 
     request_str = AV_URLS['api'] + '=' + func + from_param + from_sym + to_param + to_sym + '&apikey='
-    for key in ALPHAVANTAGE_KEYS:
 
-        response_formatted = requests.get((request_str + key)).json()
-        # frequent request
-        if 'Note' in response_formatted:
-            continue
-        # wrong command
-        elif 'Error Message' in response_formatted:
-            return
-        else:
-            return response_formatted[return_key]
-    return
+    key = ALPHAVANTAGE_KEYS[0]
```

Comment:
Why did we switch to single AV key? What happens when this key runs out of its request limit?

Reply:
No. I switched the design of usage of keys to something similar to round robin. Check lines 43 and 68.

Reply:
I get it now, thanks